pkg/ast/spl/spl.peg

```diff
@@ -320,9 +320,78 @@ AggregatorBlock <- PIPE CMD_STATS aggs:AggregationList byFields:(GroupbyBlock)?
         aggNode.BucketLimit = query.MAX_GRP_BUCKS
     }
 
+    if options != nil {
+        aggNode.StatsOptions = options.(*structs.StatsOptions)
+    } else {
+        aggNode.StatsOptions = &structs.StatsOptions {
+            Delim: " ",
+            Partitions: 1,
+            DedupSplitvals: false,
+            Allnum: false,
+        }
+    }
+
     return aggNode, nil
 }
 
+// Todo: implement the processing logic
+// Optional arguments for stats block
+StatsOptions <- option:(StatsOption)*
+{
+    //Default value
+    options := &structs.StatsOptions {
+        Delim: " ",
+        Partitions: 1,
+        DedupSplitvals: false,
+        Allnum: false,
+    }
```

Comment:
nit: maybe you should have a function that returns a default `StatsOptions`, so you don't have to define the default in both of these places